root/components/relationships-table.tt

```diff
@@ -31,10 +31,17 @@
                     [%~ END ~%]
                     <td>
                       [%~ IF rel.link.attributes.size ~%]
-                      [%~ attr_list = [];
+                      [%~ attr_list1 = []; attr_list2 = [];
+                          instrument_root = '0abd7f04-5e28-425b-956f-94789d9bcbe2';
+                          vocals_root     = 'd92884b7-ee0c-46d5-96f3-918196ba8c5b';
```

Comment:
I have no issue with changing these to use the other constants; I'd just have to look at how to get at them. Will update this PR when that's done.

As for the other suggestion, my initial version had the colon-separated form, but this immediately drew a remark (from reosarevok I think) on IRC that it looked weird. As for making it translatable, again I'd just have to know how to do that from the template (it looks like those l() calls are enough, so easy enough).